File: Java/src/main/java/ReservationResponseDto.java
import java.util.List;

public class ReservationResponseDto {
	public final String trainId;
    public final String bookingId;
    public final List<SeatDto> seats;

    public ReservationResponseDto(String trainId, List<SeatDto> seats, String bookingId) {
		this.trainId = trainId;
        this.bookingId = bookingId;
        this.seats = seats;
    }

    @Override
    public String toString() {
        return "ReservationResponseDto{" +
                "trainId='" + trainId + '\'' +
                ", bookingId='" + bookingId + '\'' +
                ", seats=" + seats +
                '}';
    }
}

File: csharp/src/ReservationResponseDto.cs
using System.Collections.Generic;

namespace TrainKata
{
    public class ReservationResponseDto
    {
        public string TrainId { get; }
        public string BookingId { get; }
        public List<Seat> Seats { get; }

        public ReservationResponseDto(string trainId, List<Seat> seats, string bookingId)
        {
            TrainId = trainId;
            BookingId = bookingId;
            Seats = seats;
        }

        public override string ToString()
        {
            return "ReservationResponseDto{" +
                   "trainId='" + TrainId + '\'' +
                   ", bookingId='" + BookingId + '\'' +
                   ", seats=" + Seats +
                   '}';
        }
    }
}
